guardrails/async_guard.py

```diff
@@ -428,22 +458,43 @@ async def _async_parse(
         Returns:
             The validated response.
         """
-        runner = AsyncRunner(
-            instructions=kwargs.pop("instructions", None),
-            prompt=kwargs.pop("prompt", None),
-            msg_history=kwargs.pop("msg_history", None),
-            api=get_async_llm_ask(llm_api, *args, **kwargs) if llm_api else None,
-            prompt_schema=self.rail.prompt_schema,
-            instructions_schema=self.rail.instructions_schema,
-            msg_history_schema=self.rail.msg_history_schema,
-            output_schema=self.rail.output_schema,
-            num_reasks=num_reasks,
-            metadata=metadata,
-            output=llm_output,
-            base_model=self.base_model,
-            full_schema_reask=full_schema_reask,
-            disable_tracer=self._disable_tracer,
-        )
+        print("\n\n\n\n\n\n")
+        print(kwargs)
+        print("\n\n\n\n\n\n")
+        if kwargs["stream"]:
+            runner = AsyncStreamRunner(
+                instructions=kwargs.pop("instructions", None),
+                prompt=kwargs.pop("prompt", None),
+                msg_history=kwargs.pop("msg_history", None),
+                api=get_async_llm_ask(llm_api, *args, **kwargs) if llm_api else None,
+                prompt_schema=self.rail.prompt_schema,
+                instructions_schema=self.rail.instructions_schema,
+                msg_history_schema=self.rail.msg_history_schema,
+                output_schema=self.rail.output_schema,
+                num_reasks=num_reasks,
+                metadata=metadata,
+                output=llm_output,
+                base_model=self.base_model,
+                full_schema_reask=full_schema_reask,
+                disable_tracer=self._disable_tracer,
+            )
```

Comment:
Are we intentionally adding support for streaming during `Guard.parse` as part of this PR?

Reply:
Maybe I'm misunderstanding the functionality that is supposed to be here. This is for the AsyncGuard, is there a better place to handle the transition to the AsyncStreamRunner vs AsyncRunner?

Reply:
No this is the right file, my comment was moreso that we previously only supported streaming in `Guard.__call__` because with parse there's nothing the stream the first time through.

Reply:
Ok yeah I understand now. Good point, removing this. Doesn't make too much sense to support here.